salt/metalk8s/containerd/installed.sls

```diff
@@ -6,7 +6,7 @@ include:
 
 Install container-selinux:
   pkg.installed:
-    - name: container-selinux
+    - source: ftp://ftp.pbone.net/mirror/ftp.scientificlinux.org/linux/scientific/7x/external_products/extras/x86_64/container-selinux-2.77-1.el7_6.noarch.rpm
```

Comment:
It's temporary?

Because relying on "random" FTP seems unreliable/unsafe.

Reply:
Yes, see #575. If we have the infrastructure in place to generate an offline repository, then this could be part of it (or, hopefully, not because by that time the upstream CentOS repositories have a more recent version of the package).